nautobot/core/views/mixins.py

```diff
@@ -234,6 +235,7 @@ class NautobotViewSetMixin(GenericViewSet, AccessMixin, GetReturnURLMixin, FormV
     serializer_class = None
     table_class = None
     notes_form_class = NoteForm
+    permission_classes = [IsAuthenticated,]
```

Comment:
`check_permissions()` on line 264 is ignoring the `permissions_classes` attribute whereas in [DRF](https://github.com/encode/django-rest-framework/blob/b25028ac8f46984459f9d9e28069b797986ae476/rest_framework/views.py#L332) check_permissions checks the specified permissions in `permission_classes`. That's why declaring `permission_classes` did not work